src/components/atom/button/types.ts

```diff
@@ -0,0 +1,10 @@
+export type ButtonProps = {
+    label: string,
+    disabled?: boolean,
+    bType: "primary" | "secondary",
+    color?: string;
+    bsize?: "small" | "middle" | "large",
+    preset?: boolean,
+    onClick?: (e: any) => void;
```

Comment:
As a standard, we should avoid using `any` type. For now, I think we can just remove `e: any` here since we don't have to pass anything on click.